src/utils/processRenderResponse.ts

```diff
@@ -65,13 +78,29 @@ export const processRenderResponse = (
            * e.g., the path will completely replaced, and re-generated (as a relative path)
            * by rollup.
            */
-          imports = `import ${INSERT_STYLE_ID} from '${INSERT_STYLE_ID}';\n`;
-          defaultExport = `${INSERT_STYLE_ID}(${out});`;
+          imports.push(`import ${INSERT_STYLE_ID} from '${INSERT_STYLE_ID}';`);
+          cssCode = `${INSERT_STYLE_ID}(${cssCode})`;
+          defaultExport = cssCode;
         } else if (!rollupOptions.output) {
-          defaultExport = out;
+          defaultExport = cssCode;
         }
 
-        return `${imports}export default ${defaultExport};\n${restExports}`;
+        const variableName = makeLegalIdentifier(
+          path.basename(fileId, path.extname(fileId)),
+        );
+
+        const codeOutput: string[] = [
+          ...imports,
+
+          `var ${variableName} = ${defaultExport};`,
```

Comment:
Could we use `const` here?

Reply:
Also the `[].join('\n')` on line 103 could be `[].join(';\n')` (would need to update other entries to not have ';' directly).